modules/nasm/2.14.02/patches/add_config_h.patch

```diff
@@ -0,0 +1,546 @@
+--- config/config.h
++++ config/config.h
+@@ -0,0 +1,543 @@
++/* config/config.h.  Generated from config.h.in by configure.  */
```

Comment:
Does this depend on any host features? If this is also copied from Tensorflow, please also add a comment for reference.